src/comm.cc

```diff
@@ -809,6 +810,7 @@ old_comm_reset_close(int fd)
 {
     if (fd >= 0) {
         commConfigureLinger(fd, OnOff::on);
+        fd_table[fd].flags.harshClosureRequested = true;
```

Comment:
We are investigating whether these assignments should be moved into commConfigureLinger() where they will use an "enabled with zero timeout" condition (instead of `true`).

Reply:
Checked and moved at fd624e60.